cmd/dcrdata/config.go

```diff
@@ -166,8 +166,8 @@ type config struct {
 	RateCertificate   string `long:"ratecert" description:"File containing DCRRates TLS certificate file." env:"DCRDATA_RATE_MASTER"`
 
 	// Links
-	MainnetLink  string `long:"mainnet-link" description:"When dcrdata is on testnet, this address will be used to direct a user to a dcrdata on mainnet when appropriate." env:"DCRDATA_MAINNET_LINK"`
-	TestnetLink  string `long:"testnet-link" description:"When dcrdata is on mainnet, this address will be used to direct a user to a dcrdata on testnet when appropriate." env:"DCRDATA_TESTNET_LINK"`
+	MainnetLink  string `long:"mainnet-link" description:"When dcrdata is on mainnet, this address will be used to direct a user to a dcrdata on mainnet when appropriate." env:"DCRDATA_MAINNET_LINK"`
+	TestnetLink  string `long:"testnet-link" description:"When dcrdata is on testnet, this address will be used to direct a user to a dcrdata on testnet when appropriate." env:"DCRDATA_TESTNET_LINK"`
```

Comment:
I think these were right.  They mean if you open testnet.dcrdata.org, it's going to use `MainnetLink` to make the link for the "Switch to Mainnet" menu item:

![image](https://user-images.githubusercontent.com/9373513/145097172-c678b30a-031c-4c8c-b2dd-180f16965ccb.png)

Vice versa for the `TestnetLink` when on a mainnet site.

Reply:
that's right. I'm sorry.